xbmc/utils/CPUInfo.cpp

```diff
@@ -419,6 +420,30 @@ CCPUInfo::CCPUInfo(void)
       }
     }
     fclose(fCPUInfo);
+    // new socs use the sysfs soc interface to describe the hardware
+    if (SysfsUtils::Has("/sys/bus/soc/devices/soc0"))
+    {
+        std::string machine, family, soc_id, revision;
```

Comment:
indentation is wrong for the whole PR, https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#3-indentation